subscriptions/subscriptions-service-http/src/main/scala/com/xebia/subscriptions/model/subscription.scala

```diff
@@ -25,13 +26,20 @@ object subscription {
 
     implicit val organizationEncoder: Encoder[OrganizationDTO] =
       Encoder[String].contramap(_.value)
+
+    implicit val organizationDecoder: Decoder[OrganizationDTO] =
+      Decoder[String].map(OrganizationDTO(_))
   }
 
   @newtype final case class RepositoryDTO(value: String)
 
   object RepositoryDTO {
+
     implicit val repositoryEncoder: Encoder[RepositoryDTO] =
       Encoder[String].contramap(_.value)
+
+    implicit val repositoryDecoder: Decoder[RepositoryDTO] =
+      Decoder[String].map(RepositoryDTO(_))
```

Comment:
If you want to save some lines, I believe you can replace these encoder and decoders by the Codec directly. Something like: 
```suggestion
    implicit val repositoryCodec: Codec[RepositoryDTO] =
      deriveCodec[RepositoryDTO]
```